src/general/MusicCategory.cs

```diff
@@ -23,24 +23,21 @@ public enum ReturnType
     public enum Transition
     {
         /// <summary>
-        ///   There is a fade between the categories
+        ///   No transition between tracks/categories
         /// </summary>
-        Fade,
-    }
+        None,
 
-    public enum TrackTransitionType
```

Comment:
I guess it kinda makes sense to combine these two enums into one. At least I can't right now think of a case where a value would be valid for one and not make sense in the other one.

Reply:
Though maybe with my latest review comment it does make sense to keep these separate?

Reply:
I think this option could also be applied to category change. If changing to same category skip fade.
I currently can't think of a transition that is only valid for tracks or categories.
Also as long, as the Transition enum is not used within a safe, this can be splitted easily.

But it's up to you.

Reply:
Okay fine, let's go with the combined enum for now, and change it in the future if needed.